binaryClock.css

```
@@ -0,0 +1,29 @@
+body{
+    background-color: #27272c;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+}
+.clock{
+    width: 222px;
+    height: 103px;
+    display: flex;
+}
+.timeUnit{
```

Better to use `kebab-case`.

```suggestion
.time-unit{
```